### DISCUSSION TITLE
所有有人搞懂1.9.1的目录同步逻辑了么

### DISCUSSION THREAD

**Discussion by LGuangming** [jxxghp/MoviePilot#2180] [讨论]
之前一直是在插件里面设置文件夹自动同步，自动创建分类文件夹，刮削，新的改了以后发现设置逻辑不太一样，多文件夹整理至各自文件夹在新的设置里面没成功，老是会同步到其他文件夹。有人有成功的经验吗

**Comment by cng10001** [comment]
你的意思是下载目录和媒体库目录对应起来吗？
是的话跟设置顺序（优先级）有关系
